FCCee/ILD_FCCee/compact/ILD_FCCee_v01/basic_defs.xml

```diff
@@ -0,0 +1,44 @@
+<define>
+
+  <constant name="world_side"             value="30*m"/>
+  <constant name="world_x"                value="world_side/2"/>
+  <constant name="world_y"                value="world_side/2"/>
+  <constant name="world_z"                value="world_side/2"/>
+
+  <constant name="ILDDetID_NOTUSED"      value="  0"/>
+
+
+  <constant name="ILDDetID_TPC"          value="  4"/>
```

Comment:
```suggestion
  <constant name="ILDDetID_TPC"          value="  4"/>
```
I think we should maybe put in a comment here as well, that states that `4` is what is expected for the TPC (at least for now)

Reply:
does this stem from the definition in 
[lcio/UTIL/ILDConf](https://github.com/iLCSoft/LCIO/blob/master/src/cpp/src/UTIL/ILDConf.cc) ?

Reply:
added comment

Reply:
Yes, I think @Victor-Schwan and me figured out that that is the source of that requirement.